deploy/operator.yaml

```diff
@@ -28,7 +28,17 @@ spec:
                   fieldPath: metadata.name
             - name: WATCH_NAMESPACE
               value: ""
+            - name: TEMPLATES_TAG
```

Comment:
I would actually leave the manifest intact and provide the fallback values in _defaults.yaml. That way we do not have to change that many files every time a version changes.

Reply:
But if we remove the variables from operator.yaml, then HCO will not be able to set correct values. (or you meant it like keep variables here and just remove values?)